.travis.yml

```diff
@@ -55,4 +54,4 @@ script:
   - bundle exec rails db:structure:load
   - bundle exec rails db:seed
   - rake ci
-  - bundle exec codeclimate-test-reporter
+  - CODECLIMATE_REPO_TOKEN=861a17b4c76ddedf9972cdaca2771f9ab0c99f4ce92298fa965d25a4be55bdbe bundle exec codeclimate-test-reporter
```

Comment:
Build fails not because of the missing token. Just open the report and see the logs. This is what you wanted - now you have full access to them ;)